statler_site/templates/play-detail-page.html

```diff
@@ -50,6 +52,26 @@
         });
       });
 
+      //function for posting review
+      function postReview() {
+        $.ajax({
+          url: "{% url 'api:play-reviews' play_id %}",
+          type: 'post',
+          data: JSON.stringify({text: $("#post_box").val()}),
+          // Set a header containing the crsf token to prevent crsf attacks
+          // django requires this and handles it automatically
+          // see https://docs.djangoproject.com/en/1.8/ref/csrf/#ajax
+          headers: {"X-CSRFToken": "{{ csrf_token }}"},
+          dataType: 'json',
+          success: postCallback
+        });
+      }
+
+      function postCallback(response) {
+        // TODO: replace this with what we want to do when a review is posted
+        alert(JSON.stringify(response));
+      }
+
```

Comment:
@aseebart, Ben won and we're sending reviews as json. I added this script for that. You could fix the formatting of the post box and fill in the TODO above.